seednodes/p2p.toml

```diff
@@ -0,0 +1,85 @@
+
+    # available transports. All defined addresses contains a single '%d' markup that is mandatory and will
+    # be replaced at runtime with the actual port value
+    [Node.Transports]
+        QUICAddress = "" # optional QUIC address. If this transport should be activated, should be in this format: /ip4/0.0.0.0/udp/%d/quic-v1
+        WebSocketAddress = "" # optional WebSocket address. If this transport should be activated, should be in this format: /ip4/0.0.0.0/tcp/%d/ws
+        WebTransportAddress = "" # optional WebTransport address. If this transport should be activated, should be in this format: /ip4/0.0.0.0/udp/%d/quic-v1/webtransport
+        [Node.Transports.TCP]
+            ListenAddress = "/ip4/0.0.0.0/tcp/%d" # TCP listen address
+            PreventPortReuse = true
+
+# P2P peer discovery section
+
+#The following sections correspond to the way new peers will be discovered
+#If all config types are disabled then the peer will run in single mode (will not try to find other peers)
+#If more than one peer discovery mechanism is enabled, the application will output an error and will not start
+
+[KadDhtPeerDiscovery]
+    #Enabled: true/false to enable/disable this discovery mechanism
+    Enabled = true
+
+    #Type represents the kad-dht glue code implementation.
+    #"legacy" will define the first implementation.
+    #"optimized" represents the new variant able to connect to multiple seeders at once. This implementation also has
+    #a built-in timer that will try to automatically reconnect to the seeders (in case the seeders recover after a
+    #premature shutdown)
+    Type = "optimized"
+
+    #RefreshIntervalInSec represents the time in seconds between querying for new peers
+    RefreshIntervalInSec = 10
+
+    #ProtocolID represents the protocol that this node will advertize to other peers
+    #To connect to other nodes, those nodes should have the same ProtocolID string
+    ProtocolID = "/erd/kad/0.2.2"
+
+    #InitialPeerList represents the list of strings of some known nodes that will bootstrap this node
+    #The address will be in a self-describing addressing format.
+    #More can be found here: https://github.com/libp2p/specs/blob/master/3-requirements.md#34-transport-agnostic
+    #Example:
+    #   /ip6/fe80::8823:6dff:fee7:f172/tcp/4001/p2p/QmYJyUMAcXEw1b5bFfbBbzYu5wyyjLMRHXGUkCXpag74Fu
+    #   /ip4/162.246.145.218/udp/4001/utp/ipfs/QmYJyUMAcXEw1b5bFfbBbzYu5wyyjLMRHXGUkCXpag74Fu
+    #
+    #If the initial peers list is left empty, the node will not try to connect to other peers during initial bootstrap
+    #phase but will accept connections and will do the network discovery if another peer connects to it
+    InitialPeerList = []
+
+    #kademlia's routing table bucket size
+    BucketSize = 100
+
+    #RoutingTableRefreshIntervalInSec defines how many seconds should pass between 2 kad routing table auto refresh calls
+    RoutingTableRefreshIntervalInSec = 300
+
+[Sharding]
```

Comment:
this could be removed, the seed node does not need to care about sharding

Reply:
added task MX-14498